### PR TITLE
#161779732 Setup tests.

### PR DESCRIPTION
**Pull request by nwashangai**
#### What does this PR do?
* Setup test using jest and enzyme
#### Description of Task to be completed?
* setup enzyme
* setup jest
* write test for App component
#### How should this be manually tested?
- npm install
- npm run test
#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
*  [ #161779732](https://www.pivotaltracker.com/story/show/161779732)
#### Screenshots (if appropriate)
* N/A
#### Questions:   
* N/A

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["@babel/env", "@babel/react"]
+  "presets": ["@babel/preset-env", "@babel/react"]
 }
```
Comment (Daymorelah) [inline .babelrc]: Add a new line after line 3, please.

File: src/setupTests.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```
Comment (Daymorelah) [inline src/setupTests.js:4]: Ditto!!!

File: src/tests/components/App.test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import { shallow } from 'enzyme';
+
+import App from '../../components/App';
+
+describe('<App />', () => {
+  it('renders three <App /> components', () => {
+    const wrapper = shallow(<App />);
+    expect(wrapper.length).toBe(1);
+  });
+});
```
Comment (Daymorelah) [inline src/tests/components/App.test.js:11]: Ditto!!..... Please, add a new line after line 11

File: src/tests/components/App.test.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import App from '../../components/App';
+
+describe('<App />', () => {
+  it('renders three <App /> components', () => {
```
Comment (Daymorelah) [inline src/tests/components/App.test.js]: Good job @nwashangai. Please, can you make the describe block more descriptive so we can know what the test suite is all about.

**Review by Daymorelah — CHANGES_REQUESTED** [review]

**Review by Daymorelah — DISMISSED** [review]

**Review by kingisaac95 — APPROVED** [review]

**Pull request merged**